api.md

```diff
@@ -105,3 +105,20 @@ Add something to the end of an array.
 ```javascript
 dispatch(arrayAdd('app.todos', 'New to do');
 ```
+
+## objectMerge
+
+merge objects with a depthLimit, deepmerge everything above the depthLimit,
```

Comment:
Depth limit is optional, please document here what the default behavior is and specify as optional in the parameter list below.